core/src/main/java/org/kiwix/kiwixmobile/core/utils/SharedPreferenceUtil.kt

```diff
@@ -129,6 +129,11 @@ class SharedPreferenceUtil @Inject constructor(val context: Context) {
     set(prefShowBookmarksFromCurrentBook) = sharedPreferences.edit {
       putBoolean(PREF_SHOW_BOOKMARKS_ALL_BOOKS, prefShowBookmarksFromCurrentBook)
     }
+  var showStorageOption: Boolean
+    get() = sharedPreferences.getBoolean(PREF_SHOW_STORAGE_OPTION, false)
```

Comment:
I think that default should be true. And after users select the default storage option then it has to store false.